examples/Extensions/UpdateSitelinkCampaignExtensionSetting.php

```diff
@@ -58,7 +56,7 @@ public static function main()
         $options = (new ArgumentParser())->parseCommandArguments([
             ArgumentNames::CUSTOMER_ID => GetOpt::REQUIRED_ARGUMENT,
             ArgumentNames::CAMPAIGN_ID => GetOpt::REQUIRED_ARGUMENT,
-            ArgumentNames::EXTENSION_FEED_ITEM_RESOURCE_NAMES => GetOpt::MULTIPLE_ARGUMENT
+            ArgumentNames::EXTENSION_FEED_ITEM_IDS => GetOpt::MULTIPLE_ARGUMENT
```

Comment:
as discussed in b/163437327, please use FEED_ITEM_IDS instead.

Reply:
Sorry, this is what I meant to do but missed it!